ast/parser.go

```diff
@@ -330,9 +330,15 @@ func (p *Parser) Parse() ([]Statement, []*Comment, Errors) {
 		s = p.save()
 
 		if imp := p.parseImport(); imp != nil {
+			if RegoRootDocument.Equal(imp.Path.Value.(Ref)[0]) {
+				// import all future keywords with the rego.v1 import
+				p.regoV1Import(imp, futureKeywords)
```

Comment:
We could omit the keywords arg of `regoV1Import` even, and let that function define what keywords are included in `V1`.
But this does what we want, so the above isn't a required change.